### PR TITLE
Test the bulk loaders with geo types.

### PR DESCRIPTION
**Pull request by BillWhite**
We test the bulk loaders with geo types.

https://issues.voltdb.com/browse/ENG-9922

### CODE REVIEW

**Comment by BillWhite** [comment]
This is all blue in Jenkins.

**Comment by hhakimi53** [comment]
Logically changes look good to me though I do not know much about the bulk loader tests. 
A suggestion, may be for one of the rows can try inserting null values for geo types, so that will cover testing with null values for geo types in bulk loader interface.

File: tests/frontend/org/voltdb/utils/TestVoltBulkLoader.java [hunk 1]
```diff
@@ -456,20 +474,22 @@ public void testNULL() throws Exception
...
-            {5,null,1,11111111," \"  \\" + Constants.CSV_NULL   + "  \"",1.10,1.11},
-            {6,Constants.CSV_NULL,1,11111111," \"  \\" + Constants.CSV_NULL  + " L \"",1.10,1.11},
-            {7,Constants.CSV_NULL,1,11111111,"  \"abc\\" + Constants.CSV_NULL + "\"  ",1.10,1.11}
+            {1,Constants.CSV_NULL,1,11111111,null,1.10,1.11, geo, geopt},
+            {2,Constants.QUOTED_CSV_NULL,1,11111111,null,1.10,1.11, geo, geopt},
+            {3,Constants.CSV_NULL,1,11111111,"  \\" + Constants.CSV_NULL + "  ",1.10,1.11, geo, geopt},
+            {4,Constants.CSV_NULL,1,11111111,"  " + Constants.QUOTED_CSV_NULL + "  ",1.10,1.11, geo, geopt},
+            {5,null,1,11111111," \"  \\" + Constants.CSV_NULL   + "  \"",1.10,1.11, geo, geopt},
+            {6,Constants.CSV_NULL,1,11111111," \"  \\" + Constants.CSV_NULL  + " L \"",1.10,1.11, geo, geopt},
+            {7,Constants.CSV_NULL,1,11111111,"  \"abc\\" + Constants.CSV_NULL + "\"  ",1.10,1.11, geo, geopt}
```
Comment (paulmartel) [inline tests/frontend/org/voltdb/utils/TestVoltBulkLoader.java]: passing null values for the geo objects at least for some rows would seem to be more in the spirit of this test case.

**Comment by paulmartel** [comment]
I just had the one comment (and I now see that Huzefa scooped me on even that one).

**Comment by BillWhite** [comment]
I added a some null values to testBlankNulls.
